bundles/org.openhab.binding.shelly/doc/READMEbeta.md

```diff
@@ -0,0 +1,127 @@
+# Shelly Binding (org.openhab.binding.shelly)
```

Comment:
I guess this file can be removed now that we are heading towards including it in 2.5.

Reply:
no, I want to keep it with information how to install beta build, how to report a bug etc.
This was the compromise: remove this stuff from the regular README and the file to doc/, because that doesn't get part of the bundle. I need to revise the content, but it will stay.

Reply:
Sorry, but we cannot accept such files in the official repo. The code that we merge will be version 2.5 and nothing else. There is no such thing as alpha, beta or 2.4.2 releases of this binding, when it is merged - there might have been some in the past, so it's ok, if you keep that as a history in your fork.
For all further discussions and sharing infos with users, the community forum is the right place.

Reply:
As mentioned, please remove this to make the PR mergeable.